### DISCUSSION TITLE
Grain boundary misorientation axis in specimen coordinate

### DISCUSSION THREAD

**Discussion by rennesqiu** [mtex-toolbox/mtex#1419] [Ask Anything] [answered]
Hi,

I would like to calculate the misorientation axis for grain boundaries (from an EBSD map) in the specimen coordinate. I found the script in the MTEX documentation for 'innerBoundaries' only, but not for all boundaries: https://mtex-toolbox.github.io/TiltAndTwistBoundaries.html

I tried some modifications on the script but I got error. See attached images please. 
<img width="413" alt="Error" src="https://user-images.githubusercontent.com/104770660/166234964-1d937b0d-2207-4012-b87f-d305512d4d15.png">
<img width="460" alt="Script" src="https://user-images.githubusercontent.com/104770660/166234974-7ae0fdaf-5bf3-48bf-8518-72620ba66ef7.png">

Could any help me out? Thanks sincerely! 

Kind regards,
Ren

**Comment by kilir** [comment, marked as answer]
Hi,
I guess it should be 
```
GB = grains.boundary('T','T')
```
Cheers,
Rüdiger

**Reply by rennesqiu** [reply]
Hi,

Thanks, it works now! 

Kind regards,
Ren
